post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -6,21 +6,17 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. You can deploy a cluster with multi-architecture compute machines by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are only available on AWS or Azure installer-provisioned infrastructures and Bare Metal user-provisioned infrastructures with x86_64 control plane machines. 
 
-[WARNING]
-====
-The multi-architecture compute machines Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
-====
+== Creating a cluster with multi-architecture compute machine on Azure 
 
-The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the desired amount of `arm64` virtual machines (VM). This section also shows how to upgrade your existing cluster to a cluster that supports multi-architecture compute machines. Clusters with multi-architecture compute machines are only available on Azure installer-provisioned infrastructures with `x86_64` control plane machines. 
+To deploy a Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster with the multi-architecture installer binary. For more information on Azure installations, refer to xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations]. You can then add a ARM64 compute machine set to your cluster to create a cluster with multi-architecture compute machines. 
 
-:FeatureName: {product-title} clusters with multi-architecture compute machines on Azure installer-provisioned infrastructure installations
-include::snippets/technology-preview.adoc[leveloffset=+1]
+The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the desired amount of `arm64` virtual machines (VM). 
```

Comment:
```suggestion
The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the amount of `arm64` virtual machines (VM) that you need. 
```
In the first sentence, consider replacing "...with the `arm64` boot image" to either "...by using the `arm64` boot image" or "...that uses the `arm64` boot image", depending on the intended meaning.

Reply:
Both SSG and ISG suggest using another term besides "desired" in technical documentation.

Reply:
Yeah I like that better as well. Thank you!